plugins/generator-1.15.2/forge-1.15.2/potion.definition.yaml

```diff
@@ -1,18 +1,13 @@
-templates:
-  - template: potion.java.ftl
-    name: "@SRCROOT/@BASEPACKAGEPATH/potion/@NAMEPotion.java"
-
-localizationkeys:
-  - key: effect.@registryname
-    mapto: effectName
-  - key: item.minecraft.potion.effect.@registryname
-    mapto: name
-  - key: item.minecraft.splash_potion.effect.@registryname
-    mapto: name
-    prefix: "Splash "
-  - key: item.minecraft.tipped_arrow.effect.@registryname
-    mapto: name
-    prefix: "Arrow of "
-  - key: item.minecraft.lingering_potion.effect.@registryname
-    mapto: name
-    prefix: "Lingering "
\ No newline at end of file
+templates:
+  - template: potion.java.ftl
+    name: "@SRCROOT/@BASEPACKAGEPATH/potion/@NAMEPotion.java"
+
+localizationkeys:
+  - key: item.minecraft.potion.effect.@registryname
+    mapto: potionName
+  - key: item.minecraft.splash_potion.effect.@registryname
+    mapto: splashName
+  - key: item.minecraft.tipped_arrow.effect.@registryname
```

Comment:
These are swapped, same for 1.16.5

Reply:
wdym? what is swapped?

Reply:
Oh I see what you meant, I just fixed it...